packages/boxel-ui/addon/src/components/input/index.gts

```diff
@@ -288,11 +284,25 @@ export default class BoxelInput extends Component<Signature> {
       }
 
       .search {
+        --search-input-color: var(
+          --boxel-input-search-color,
+          var(--boxel-light)
+        );
+
+        --search-input-background-color: var(
+          --boxel-input-search-background-color,
+          var(--boxel-dark)
+        );
+        --search-input-placeholder-color: var(
+          --boxel-input-search-placeholder-color,
+          var(--boxel-light)
```

Comment:
optionally, you can say `inherit` or `currentColor` for the placeholder color fallback value and it will be the same color as your input text color without needing to specify it